### PR TITLE
feat/homepage

### PR DESCRIPTION
**Pull request by adith-a-danthi**
## Overview

- Add `tidle-ui` and `fontAwesome`
- Add `react-router`
- Add `Home` and `404` pages
- Add `CategoryCard` and `VideoCard` components

### CODE REVIEW

File: src/views/Home/Home.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/* Home */
+.category-list {
+  display: grid;
+  gap: 2rem;
+  /* grid-template-columns: repeat(auto-fit, minmax(300px, 1fr)); */
```
Comment (Tanay-J) [inline src/views/Home/Home.css]: You may this remove comment

**Review by Tanay-J — APPROVED** [review]
- Nice UI. 
- Well structured files and folders. 
- Good use of reset.css.
Approving the changes

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+              <span className="fa-brands fa-github fa-xl"></span>
+            </a>
+            <a
+              href="https://linkedin.com/in/adith-a-danthi/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <span className="fa-brands fa-linkedin fa-xl"></span>
+            </a>
+            <a href="http://adith.dev" target="_blank" rel="noopener noreferrer">
```
Comment (chanchal16) [inline src/components/Footer.jsx:19]: Liked the use of target blank indicates a good UX experience.

File: src/views/Home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+              title="Super Nintendo"
+              image="https://images.unsplash.com/photo-1602029908656-b54d40a76ad8?ixlib=rb-1.2.1&raw_url=true&q=80&fm=jpg&crop=entropy&cs=tinysrgb&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170"
+              channel="@Nintendo"
+            />
+            <VideoCard
+              title="MacBook Air"
+              image="https://images.unsplash.com/photo-1548092372-0d1bd40894a3?ixlib=rb-1.2.1&raw_url=true&q=80&fm=jpg&crop=entropy&cs=tinysrgb&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170"
+              channel="@Apple"
+            />
+            <VideoCard
```
Comment (chanchal16) [inline src/views/Home/Home.jsx:57]: Suggestion: Instead of writing four different cards, you can store the info in a JSON file and map through it.

**Review by chanchal16 — APPROVED** [review]
Left one suggestion other than that looks pretty good. Good to go ahead!

File: src/views/Home/Home.jsx (hunk 3 above)
Comment (adith-a-danthi) [reply src/views/Home/Home.jsx:57]: Will do that with the actual data. 👍

**Pull request merged**